src/psyclone/tests/psyir/nodes/intrinsic_call_test.py

```diff
@@ -434,15 +434,10 @@ def test_reference_accesses_bounds(operator, fortran_reader):
     psyir = fortran_reader.psyir_from_source(code)
     schedule = psyir.walk(Assignment)[0]
 
-    # By default, the access to 'a' should not be reported as read,
-    # but the access to b must be reported:
+    # The access to 'a' should be reported as 'NO DATA ACCESS' as its
+    # actual data is not accessed.
     vai = VariablesAccessInfo(schedule)
-    assert str(vai) == "b: READ, n: WRITE"
-
-    # When explicitly requested, the access to 'a' should be reported:
-    vai = VariablesAccessInfo(schedule,
-                              options={"COLLECT-ARRAY-SHAPE-READS": True})
-    assert str(vai) == "a: READ, b: READ, n: WRITE"
+    assert str(vai) == "a: NO DATA ACCESS, b: READ, n: WRITE"
```

Comment:
This formatting makes me think it would be clearer as a single string with no whitespaces, e.g. `NO_DATA_ACCESS`. What do you think?

Reply:
Yes, I did find myself writing that sometimes. I'll change it.